crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -230,7 +230,7 @@ $env.config = {
     footer_mode: "25" # always, never, number_of_rows, auto
     float_precision: 2 # the precision for displaying floats in tables
     buffer_editor: "" # command that will be used to edit the current line buffer with ctrl+o, if unset fallback to $env.EDITOR and $env.VISUAL
-    use_ansi_coloring: true
+    ansi_coloring: enabled # enabled, disabled or auto
```

Comment:
I think this would be better as `true`, `false`, `auto` to stay consistent with the rest of the config options.

Reply:
makes sense.